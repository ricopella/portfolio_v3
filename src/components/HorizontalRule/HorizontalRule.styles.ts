import styled from '@emotion/styled'

const HorizontalRule = styled.div`
  border: 0;
  height: 0.0625rem;
  background-image: linear-gradient(
    to right,
    rgba(0, 0, 0, 0),
    rgba(0, 0, 0, 0.25),
    rgba(0, 0, 0, 0)
  );
  margin-top: 5%;
`

export default {
  HorizontalRule,
}
