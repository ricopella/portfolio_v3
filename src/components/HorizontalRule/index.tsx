import React from 'react'
import ScrollReveal from '../../components/ScrollReveal'
import SlideUp from '../SlideUpElement'
import Styled from './HorizontalRule.styles'

const HorizontalRule = () => (
  <ScrollReveal>
    <SlideUp>
      <Styled.HorizontalRule />
    </SlideUp>
  </ScrollReveal>
)

export default HorizontalRule
