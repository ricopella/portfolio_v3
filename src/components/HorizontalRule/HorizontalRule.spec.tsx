import HorizontalRule from './index'
import React from 'react'
import renderer from 'react-test-renderer'

describe('<HorizontalRule />', () => {
  it('should match HorizontalRule snapshot', () => {
    const internalTree = renderer.create(<HorizontalRule />).toJSON()

    expect(internalTree).toMatchSnapshot()
  })
})
