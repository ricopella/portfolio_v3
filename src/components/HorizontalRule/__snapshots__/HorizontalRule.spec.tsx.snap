// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HorizontalRule /> should match HorizontalRule snapshot 1`] = `
<div
  data-sal="slide-up"
  data-sal-delay="200"
  data-sal-easing="ease"
>
  <div
    style={
      Object {
        "opacity": 0,
        "transform": "translateY(20px) translateZ(0)",
      }
    }
  >
    <div
      className="css-1a4pyaz"
    />
  </div>
</div>
`;
